doc/classes/HTTPRequest.xml

```diff
@@ -1,13 +1,43 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="HTTPRequest" inherits="Node" category="Core" version="3.2">
 	<brief_description>
-		A node with the ability to send HTTP requests.
+		A node with the ability to send HTTP(S) requests.
 	</brief_description>
 	<description>
 		A node with the ability to send HTTP requests. Uses [HTTPClient] internally.
```

Comment:
If you're changing the brief description to "HTTP(S)" then perhaps this line should be "HTTP(S)" or perhaps "HTTP or HTTPS".

Reply:
I'm not an expert on HTTP, but when using HTTPS do we actual call those requests "HTTPS requests", or are they still "HTTP requests" that happen to use the secure protocol? (i.e., are those requests the same, or are they different and thus should be explicitly listed as two types of requests supported by this node)

Reply:
HTTPS requests are the same as HTTP requests. They just happen to use encryption while transferring data. 

You do not need to change your request in any way except for writing `https://...` instead of `http://...` into the url when making such a request using a http library/client. The underlying implementation is expected to handle everything else from making the TCP connection through handling the encryption of the data to returning the response.

In practice a http library or client that claims to be able to make "HTTP Requests" is expected to have HTTPS support implemented even if it isn't explicitly documented. Otherwise it would be unusable for most real-world scenarios.

In conclusion: any of the 3 wordings mentioned above in this thread should be correct in this context. Even the plain old "HTTP Request" term.